web/src/services/api/projectApi.ts

```diff
@@ -122,9 +122,26 @@ export default () => {
     return { deletedProjects, ...rest };
   }, []);
 
-  const useProjectAliasCheckLazyQuery = useCallback(() => {
-    return useLazyQuery(CHECK_PROJECT_ALIAS);
-  }, []);
+const [fetchCheckAlias] = useLazyQuery(CHECK_PROJECT_ALIAS);
+
+const checkAlias = useCallback(
+  async (alias?: string) => {
+    if (!alias) return null;
+
+    const { data, error } = await fetchCheckAlias({
+      variables: { alias }
+    });
+
+    if (error) {
+      console.error("Failed to check alias:", error);
+      return null;
+    }
+
+    return data?.checkProjectAlias;
+  },
+  [fetchCheckAlias]
+);
```

Comment:
I just gave an general example, not 100% implemented for this case, please consider error handling and proper return value.